lib/schema.js

```diff
@@ -0,0 +1,109 @@
+const _ = require('lodash')
+const Joi = require('joi')
+const { SORT_DIRECTIONS } = require('./sort-pull-requests')
+
+const category = Joi.object().keys({
+  title: Joi.string().required(),
+  label: Joi.string()
+})
+
+const searchReplace = Joi.object().keys({
+  search: Joi.string()
+    .regex(/^\/.+\/[gmixXsuUAJ]*$|.+/)
```

Comment:
Given this has `.+`, I wonder what benefit there is to having the first part of the pattern (or defining any `regex()` at all?)

Reply:
None, I was toying with the idea :laughing:

Reply:
😆